### PR TITLE
✨ description of checks migrated

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
doc

* **What is the current behavior?** (You can also link to an open issue here)
no doc

* **What is the new behavior (if this is a feature change)?**
new doc

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

**Review by inferno-chromium — COMMENTED** [review]
Any reason why we are creating seperate file. it is best to modify the main checks.yaml and add the risk field there. also, it will be easy to review the diff w.r.t content.

**Comment by laurentsimon** [comment]
> Any reason why we are creating seperate file. it is best to modify the main checks.yaml and add the risk field there. also, it will be easy to review the diff w.r.t content.

yes: just temporarily to avoid having some checks with `risk` field and some checks without. The file will be renamed `checks.yaml` once the migration is over; and we will update the script to output the `checks.md` as well in the last PR.

**Comment by inferno-chromium** [comment]
> > Any reason why we are creating seperate file. it is best to modify the main checks.yaml and add the risk field there. also, it will be easy to review the diff w.r.t content.
> 
> yes: just temporarily to avoid having some checks with `risk` field and some checks without. The file will be renamed `checks.yaml` once the migration is over; and we will update the script to output the `checks.md` as well in the last PR.

i am fine with an empty field value for risk in those checks or even not having the field at all, isnt that better ? for checks.md, fine to have it without risk field in autogeneration, can be added once you have all risks added.

**Comment by azeemshaikh38** [comment]
> > > Any reason why we are creating seperate file. it is best to modify the main checks.yaml and add the risk field there. also, it will be easy to review the diff w.r.t content.
> > 
> > 
> > yes: just temporarily to avoid having some checks with `risk` field and some checks without. The file will be renamed `checks.yaml` once the migration is over; and we will update the script to output the `checks.md` as well in the last PR.
> 
> i am fine with an empty field value for risk in those checks or even not having the field at all, isnt that better ? for checks.md, fine to have it without risk field in autogeneration, can be added once you have all risks added.

+1. Lets have this in the same `checks.yaml` file.

**Comment by laurentsimon** [comment]
> > > > Any reason why we are creating seperate file. it is best to modify the main checks.yaml and add the risk field there. also, it will be easy to review the diff w.r.t content.
> > > 
> > > 
> > > yes: just temporarily to avoid having some checks with `risk` field and some checks without. The file will be renamed `checks.yaml` once the migration is over; and we will update the script to output the `checks.md` as well in the last PR.
> > 
> > 
> > i am fine with an empty field value for risk in those checks or even not having the field at all, isnt that better ? for checks.md, fine to have it without risk field in autogeneration, can be added once you have all risks added.
> 
> +1. Lets have this in the same `checks.yaml` file.

done, ptal

File: checks/checks.md [hunk 1]
```diff
@@ -11,28 +11,36 @@ If you have ideas for things to add, or new ways to detect things,
...
+Branch protection allows defining rules to enforce certain workflows for branches, such as requiring a review or passing certain status checks.
+Branch protection ensures compromised contributors cannot  intentionally inject malicious code. A low score is therefore considered `High` risk.
+This check determines if the default and release branches are protected with GitHub's branch protection settings. The check only works when the token has [Admin access](https://github.community/t/enable-branch-protection-get-api-without-admin/14197) to the repository. This check determines if the default and release branches are protected.
```
Comment (inferno-chromium) [inline checks/checks.md:43]: ;linkify GitHub's branch protection settings to the right doc.

File: checks/checks.md [hunk 2]
```diff
@@ -11,28 +11,36 @@ If you have ideas for things to add, or new ways to detect things,
...
 
 **Remediation steps**
 - There is *NO* remediation work needed here. This is just to indicate your project activity and maintenance commitment.
 
 ## Automatic-Dependency-Update 
 
-This check tries to determine if a project has dependencies automatically updated. The checks looks for [dependabot](https://dependabot.com/docs/config-file/) or [renovatebot](https://docs.renovatebot.com/configuration-options/). This check only looks if  it is enabled and does not ensure that it is run and pull requests are merged. 
+This check tries to determine if a project has dependencies automatically updated.
+Not updating dependencies makes a project vulnerable to known flaws and prone to attacks.  A low score is therefore considered `High` risk.
+The checks looks for [dependabot](https://dependabot.com/docs/config-file/) or [renovatebot](https://docs.renovatebot.com/configuration-options/). This check only looks if  it is enabled and does not ensure that it is run and pull requests are merged.
```
Comment (inferno-chromium) [inline checks/checks.md:25]: two spaces between "if" and "it is enabled"

**Review by inferno-chromium — APPROVED** [review]
A lot of high risks, fine for now. when you add all the risks, we can discuss in a group to double check and discuss risk values.

**Review by azeemshaikh38 — APPROVED** [review]

**Pull request merged**